src/tests/functional/plugin/cpu/test_utils/cpu_test_utils.cpp

```diff
@@ -346,6 +344,18 @@ void CheckNodeOfTypeCount(InferenceEngine::ExecutableNetwork &execNet, std::stri
 
     ASSERT_EQ(expectedCount, actualNodeCount) << "Unexpected count of the node type '" << nodeType << "' ";
 }
+
+void CheckNodeOfTypeCount(ov::runtime::CompiledModel &execNet, std::string nodeType, size_t expectedCount) {
+    std::shared_ptr<const ov::Model> function = execNet.get_runtime_model();
+    CheckNodeOfTypeCountImpl(function, nodeType, expectedCount);
+}
+
+void CheckNodeOfTypeCount(InferenceEngine::ExecutableNetwork &execNet, std::string nodeType, size_t expectedCount) {
```

Comment:
Perhaps checkNumberOfNodesWithType is clearer?

Reply:
renamed